pages/omeneses/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Carousel</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <section class="carousel">
+    <div class="heading-title">
+                <h1>our <span class="heading-title--orange-font">hotels</span></h1>
+    </div>
+    <div class="card-container">  
+      <div>
+          <button class="button-arrow">
+              <img src="./chevron_left.png" class="button-arrow__image">
+         </button>
+      </div>    
+      <div class="card">
+        <div>
+          <img src="Cuba.jpg" alt="card1" class="card__image" />
+        </div>
+        <h3 class="card__caption">Cuba</h3>
+        <ul class="card__hotels-list">
+          <li>Hotel 1</li>
+          <li>Hotel 2</li>
+          <li>Hotel 3</li>
+        </ul>
+        <div class="card__cta">
+            <button class="button-cta">More ></button>
+        </div>
+      </div>
+      <div class="card">
+        <div>
+            <img src="New York.jpg" alt="card1" class="card__image" />
+        </div>
+        <h3 class="card__caption">New York</h3>
+        <ul class="card__hotels-list">
+          <li>Hotel x</li>
+          <li>Hotel y</li>
+          <li>Hotel z</li>
+        </ul>
+        <div class="card__cta">
+            <button class="button-cta">More ></button>
+        </div>
+      </div>
+      <div class="card">
+        <div>
+          <img src="Paris.jpg" alt="card3" class="card__image" />
+        </div>
+        <h3 class="card__caption">Paris</h3>
+        <ul class="card__hotels-list">
+          <li>Hotel 4</li>
+          <li>Hotel 5</li>
+          <li>Hotel 6</li>
+        </ul>
+        <div class="card__cta">
+            <button class="button-cta">More ></button>
+        </div>
+      </div>
+      <div>
+          <button class="button-arrow">
+              <img src="./chevron_right.png" class="button-arrow__image" >
+         </button>
```

Comment:
Do you think this arrow could be a separate element of the carousel?

Reply:
You're right, it shouldn't.  I'll fix it.